### ISSUE TITLE
LICENSE.md disagrees with bootstrap-slider.js on license used

### ISSUE THREAD

**Issue by yuethomas** [seiyria/bootstrap-slider#648]
Hi,

I just noticed that LICENSE.md says that bootstrap-slider is licensed under MIT, but the header information in bootstrap-slider.js (in both dist/ and src/) say that bootstrap-slider is licensed under Apache.  I'd like to make sure which license the software is actually using to avoid any confusion.

Thanks!

**Comment by seiyria** [comment]
I believe we re-licensed from MIT to apache, but that particular file was not updated. File headers used to not be there, so those are most definitely newer.

**Comment by rovolution** [comment]
Thanks for pointing this out @yuethomas! we are on MIT but will make sure to update the header generation code in our build step

**Comment by yuethomas** [comment]
Thank you for the history. I think it could actually have been the other way around; I checked the blame for src/bootstrap-slider.js, and the section on Apache was included in the initial commit from when it was forked from Stefan Petre's original code (in 2013), and has not been updated since: 
https://github.com/seiyria/bootstrap-slider/commit/e5726a2c8746bdc7e459dcbf1f4cd912caad435b .

On the other hand, the LICENSE.md file was not created until 2014, and there was a specific commit on Jan 31, 2015 that switched it from Apache 2.0 to MIT:
https://github.com/seiyria/bootstrap-slider/commit/457979e6b35853109a80950febf43cccf35381bb .

So if I'm understanding both of these pieces of information correctly, it means that bootstrap-slider was originally using Apache 2.0, but switched to MIT in 2015 and the file header information in src/ wasn't updated to match.

@rovolution chimed in before I finished writing this comment and confirmed that it's using MIT. Thanks to you both for your help!
